backend/src/main/java/com/redhat/cloud/notifications/templates/Rhosak.java

```diff
@@ -29,7 +33,9 @@ public TemplateInstance getTitle(String eventType, EmailSubscriptionType type) {
         if (eventType.equals(ACTION_REQUIRED)) {
             return Templates.actionRequiredTitle();
         }
-
+        if (eventType.equals(ACTION_REQUIRED)) {
```

Comment:
```suggestion
        if (eventType.equals(SCHEDULED_UPGRADE)) {
```
I think you mean this here

Reply:
Good catch. Thanks